src/Components/Teammember.js

```diff
@@ -0,0 +1,78 @@
+import styled from "styled-components";
+
+/**
+ * The Team Member Component
+ * @author [Sam](https://github.com/Samm96)
+ *
+ * React template to create team members.
+ */
+
+// note: remember to replace values with Theme.js props
+
+const Container = styled.div.attrs(() => ({ tabIndex: 0 }))`
+  width: 351px;
+  display: flex;
+
+  &.teammember__text-container {
+    margin: 0 0 0 20px;
+    max-width: 205px;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  &.teammember__name-container {
+    width: fit-content;
+  }
+`;
+
+const Photo = styled.img`
+  border-radius: 50%;
+  width: 126px;
+  height: 126px;
+  object-fit: cover;
+  background-position: center;
```

Comment:
I think this only applies to the _background-image_ property. Should it be _object-position_ instead?